taskmanager/features/task.feature

```diff
@@ -9,3 +9,12 @@
       When I create a task named "Add switch language button"
       Then there should be "1" tasks with "TODO" status
 
+    Scenario: Assigned unassigned task
```

Comment:
`Scenario: Possibility to assign a task`

Reply:
What about assigning already assigned task (like reassign)? In those cases scenario will be different, so  I propose two scenarios:
1) `Scenario: Assign an unassigned task` or `Scenario: Assign a task` 
2) `Scenario: Assign an assigned task` or `Scenario: Reassign a task`